pom.xml

```diff
@@ -223,8 +223,8 @@
     <testsThreadCount>4</testsThreadCount>
 
     <slf4j.version>2.0.7</slf4j.version>
-    <junit.jupiter.version>5.10.1</junit.jupiter.version>
-    <jacoco.version>0.8.11</jacoco.version>
+    <junit-bom.version>5.11.2</junit-bom.version>
+    <jacoco.version>0.8.12</jacoco.version>
```

Comment:
According to the comments on stack overflow:

> upgrading jacoco-plugin from 0.8.1 to 0.8.4 did the job

But we already have 0.8.11, which is newer than both, so I guess this won't fix the problem.

We can upgrade anyway, Ozone has been using 0.8.12 for 6 months now (HDDS-10660).

Reply:
I am also aware that.  Just wonder they might have a better fix in a newer version.